Packs/ZeroTrustAnalyticsPlatform/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,39 @@
+
+#### Incident Fields
+- Deprecated **ZTAP Input Tag**
+- **ZTAP Alert Link**
+
+#### Integrations
+##### ZeroTrustAnalyticsPlatform
+- Updated the Docker image to: *demisto/python3:3.10.4.27798*.
+- Add url field to new incidents
+- Simplify comments to use text instead of JSON
+- Remove audit log ingestion
+- Change default to mirror in both directions by default
+- Update branding
+
+#### Layouts
+##### ZTAP Alert
+- Add ZTAP Alert Link
+- Remove unused Escalate Organization
+
+#### Mappers
+##### ZeroTrustAnalyticsPlatform - Incoming Mapper
+- Add ZTAP Alert Link
+
+#### Playbooks
+##### ZTAP Alert
+- Add ZTAP Alert Link
```

Comment:
@vimpadawan Couldn't find anywhere in the playbook anything related to the new alert link field. I did see that you removed one task which runs the `ZTAPBuildTimeline` script though.

Reply:
`ZTAPBuildTimeline` was removed from the playbook, the alert was not added to the playbook. I updated the release notes.